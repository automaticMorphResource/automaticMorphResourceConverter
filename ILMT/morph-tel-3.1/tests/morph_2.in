1	koVttaMpE	unk	
1	neVlalawarabadi	unk	
1	o	unk	
1	wIgaku	unk	
1	velAde	unk	
1	gummadilo	unk	
1	axBuwa	unk	
1	ORaXAlu	unk
1	unnAyi	unk	
1	.	unk	
