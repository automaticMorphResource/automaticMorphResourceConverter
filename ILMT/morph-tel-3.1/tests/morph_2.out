1	koVttaMpE	unk	<fs af='koVttaM,n,,sg,,,pE,pE'>
2	neVlalawarabadi	unk	<fs af='neVlalawarabadi,avy,,,,,,'>
3	o	unk	<fs af='o,avy,,,,,,'>|<fs af='o,avy,,,,,0,0_avy'>
4	wIgaku	unk	<fs af='wIga,n,,sg,,,ki,ki'>
5	velAde	unk	<fs af='velAdu,v,any,any,any,,e,e'>|<fs af='velAdu,v,any,any,any,,i,i_e'>
6	gummadilo	unk	<fs af='gummadi,n,,sg,,,lo,lo'>
7	axBuwa	unk	<fs af='axBuwaM,n,,sg,,o,ti,ti'>
8	ORaXAlu	unk	<fs af='ORaXaM,n,,pl,,d,0,0'>|<fs af='ORaXaM,n,,pl,,,vu,vu'>
9	unnAyi	unk	<fs af='uMdu,v,n,pl,3,,A,A'>
10	.	unk	<fs af='&dot;,punc,,,,,,'>
